mysql.go

```
@@ -137,6 +138,14 @@ func columnMapper(column *rel.Column) (string, int, int) {
 	}
 }
 
+func dropKeyMapper(typ rel.KeyType) string {
+	if typ == rel.ForeignKey {
+		return "FOREIGN KEY"
+	}
+
+	return ""
```

is this means not supporting other key type?
should we make it clearer rather than returning invalid sql syntax?

---

> should we make it clearer rather than returning invalid sql syntax?

yes. 

It might be better to have a more specific function like `dropForegnKeyMapper`.

---

maybe we can just panic instead of return ""?

---

updated.